dataland-frontend/src/components/structure/TheHeader.vue

```diff
@@ -12,10 +12,12 @@
       </span>
     </div>
 
-    <div class="col-1 col-offset-3 max-w-full max-h-full">
+    <div class="col-4 flex justify-content-end" v-if="!sampleData">
```

Comment:
Anstatt Offset und dann voller Breite, nimmst du den ganzen Container in Anspruch, verschiebst aber die Inhalte an das rechte Ende, korrekt?